app/EatUp_Client/app/src/main/java/com/github/eatup_client/model/Allergy.java

```diff
@@ -0,0 +1,49 @@
+package com.github.eatup_client.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Allergy implements Serializable {
+
+    @SerializedName("id")
+    @Expose
+    private String id;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("img_id")
+    @Expose
+    private String img_id;
```

Comment:
Caution! This is no longer supported in server v1.0.0. This is not throwing any errors?